src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -139,6 +139,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->defaultValue('error_controller')
                 ->end()
                 ->booleanNode('handle_all_throwables')->info('HttpKernel will handle all kinds of \Throwable')->end()
+                ->arrayNode('map_request_payload')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('status_code_on_error')
```

Comment:
is there a usecase that requires a different default?

Reply:
As described in #50992, sometimes we want use 400 instead of 422 in application-global. So this configuration node supports to change the HTTP status code on error globally.

Reply:
BTW, this is actually dupe of #50767, so I'll close this pull request now.